underfs/hdfs/pom.xml

```diff
@@ -73,10 +73,6 @@
         <version>2.18.1</version>
         <configuration>
           <argLine>-Djava.security.krb5.realm= -Djava.security.krb5.kdc=</argLine>
```

Comment:
I think you can remove this entire plugin if the integration test are all in the new module.